packages/sanes-chrome-extension/src/utils/test/dom.tsx

```diff
@@ -31,8 +31,28 @@ export const createDom = (
         </ToastProvider>
       </MedulasThemeProvider>
     </Provider>,
-  ) as any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  ) as any;
 
-export const expectRoute = (store: Store, route: string): void => {
-  expect(store.getState().router.location.pathname).toBe(route);
+export const click = (element: Element): void => {
+  TestUtils.act(() => TestUtils.Simulate.click(element));
+};
+
+export const input = (field: Element, value: any): void => {
+  TestUtils.act(() => {
```

Comment:
Can you give a spin through the tests and update them to use those helpers?

Reply:
I'm doing that for #260, besides adding the new tests.

I'm not sure if it would be worth it to refactor all the existing tests on this PR because by adding the tests documented on the spreadsheet some of them become obsolete or redundant.

Maybe I could add to this PR the refactor of Welcome and Login, that are short and to the point, and showcase this new methods.

Let me know the best course of action.

Reply:
Sounds good, continue doing that in #260. I will merge this PR now. Good job.